rllib/agents/callbacks.py

```diff
@@ -568,3 +568,133 @@ def on_train_result(self, *, trainer, result: dict, **kwargs) -> None:
         # trainer._iteration as a parameter to on_learn_on_batch() call.
         RE3UpdateCallbacks._step = result["training_iteration"]
         super().on_train_result(trainer=trainer, result=result, **kwargs)
+
+
+class NovelDMetricsCallbacks(DefaultCallbacks):
```

Comment:
I think this should be part of the NovelD package. Could we move this class into utils/exploration/noveld.py?

Reply:
> I think this should be part of the NovelD package. Could we move this class into utils/exploration/noveld.py?

I get some persistent circular imports here coming from the `DefaultCallbacks` import into `noveld.py` I guess